src/utils/git-ignore.ts

```diff
@@ -27,3 +30,7 @@ export function modifyGitIgnore(projectDir: string) {
     pathsToIgnore.forEach(path => cli.log('  ' + path));
   }
 }
+
+function cdsPathsToIgnore() {
```

Comment:
This could be a `const` instead of a `function` (unless you're concerned that someone ninja pushes values into it).